README.md

```diff
@@ -128,15 +128,15 @@ timetrace start <PROJECT KEY>
 
 **Arguments:**
 
-|Argument|Description|
-|-|-|
-|`PROJECT KEY`|The key of the project.|
+| Argument      | Description             |
+| ------------- | ----------------------- |
+| `PROJECT KEY` | The key of the project. |
```

Comment:
Oh wow, thanks for beautifying those tables! 😄

Reply:
Honestly, I don't know anything about this! It's not possible that I had accidentally merged someone else's work when I did a `git fetch upstream`, is it?

My other guess is that my editor somehow interpreted these as snippets and automatically formatted the markdown as a :sweat_smile:

Reply:
> Honestly, I don't know anything about this! It's not possible that I had accidentally merged someone else's work when I did a git fetch upstream, is it?

No, you would have to fetch that particular contributor's fork and merge their commits into your branch - and if you do so, that contributor still appears as the author of those commits in your PR. Which editor are you using? 😄

Reply:
I see, that makes sense. And day by day I see more of the genius that went into creating git!

I'm using Neovim, specifically [Lunarvim ](https://github.com/ChristianChiarulli/LunarVim) which provides an IDE layer on top of Neovim.